db/data-strict.txt

```diff
@@ -5,235 +5,235 @@ Gate (feminine): porta portae portae portam portā
+Hear: audītus eram audītus erās audītus erat audītī erāmus audītī erātis audītī erant
+Take: captus eram captus erās captus erat captī erāmus captī erātis captī erant
 
 --- Future Perfect ---
-Praise: laudatus ero laudatus eris laudatus erit laudati erimus laudati eritis laudati erunt
-Remind: monitus ero monitus eris monitus erit moniti erimus moniti eritis moniti erunt
-Lead: ductus ero ductus eris ductus erit ducti erimus ducti eritis ducti erunt
-Hear: auditus ero auditus eris auditus erit auditi erimus auditi eritis auditi erunt
-Take: captus ero captus eris captus erit capti erimus capti eritis capti erunt
+Praise: laudātus erō laudātus eris laudātus erit laudātī erimus laudātī eritis laudātī erunt
+Remind: monitus erō monitus eris monitus erit monitī erimus monitī eritis monitī erunt
+Lead: ductus erō ductus eris ductus erit ductī erimus ductī eritis ductī erunt
+Hear: audītus erō audītus eris audītus erit audītī erimus audītī eritis audītī erunt
+Take: captus erō captus eris captus erit captī erimus captī eritis captī erunt
 
 === Verbs, Subjunctive Active ===
 
 --- Present ---
-Praise: laudem laudes laudet laudemus laudetis laudent
-Remind: moneam moneas moneat moneamus moneatis moneant
-Lead: ducam ducas ducat ducamus ducatis ducant
-Hear: audiam audias audiat audiamus audiatis audiant
-Take: capiam capias capiat capiamus capiatis capiant
+Praise: laudem laudēs laudet laudēmus laudētis laudent
+Remind: moneam moneās moneat moneāmus moneātis moneant
+Lead: dūcam dūcās dūcat dūcāmus dūcātis dūcant
+Hear: audiam audiās audiat audiāmus audiātis audiant
+Take: capiam capiās capiat capiāmus capiātis capiant
 
 --- Imperfect ---
-Praise: laudarem laudares laudaret laudaremus laudaretis laudarent
-Remind: monerem moneres moneret moneremus moneretis monerent
-Lead: ducerem duceres duceret duceremus duceretis ducerent
-Hear: audirem audires audiret audiremus audiretis audirent
-Take: caperem caperes caperet caperemus caperetis caperent
+Praise: laudārem laudārēs laudāret laudārēmus laudārētis laudārent
+Remind: monērem monērēs monēret monērēmus monērētis monērent
+Lead: dūcerem dūcerēs dūceret dūcerēmus dūcerētis dūcerent
+Hear: audīrem audīrēs audīret audīrēmus audīrētis audīrent
+Take: caperem caperēs caperet caperēmus caperētis caperent
 
 --- Perfect ---
-Praise: laudaverim laudaveris laudaverit laudaverimus laudaveritis laudaverint
-Remind: monuerim monueris monuerit monuerimus monueritis monuerint
-Lead: duxerim duxeris duxerit duxerimus duxeritis duxerint
-Hear: audiverim audiveris audiverit audiverimus audiveritis audiverint
-Take: ceperim ceperis ceperit ceperimus ceperitis ceperint
+Praise: laudāverim laudāverīs laudāverit laudāverīmus laudāverītis laudāverint
+Remind: monuerim monuerīs monuerit monuerīmus monuerītis monuerint
+Lead: dūxerim dūxerīs dūxerit dūxerīmus dūxerītis dūxerint
+Hear: audīverim audīverīs audīverit audīverīmus audīverītis audīverint
+Take: cēperim cēperīs cēperit cēperīmus cēperītis cēperint

```

Comment:
The sources I've looked at show no macrons on the i's, so to speak. Hence all of these become like: `laudāverim laudāveris laudāverit laudāverimus laudāveritis laudāverint`

Thoughts?

Reply:
You’re right, I got a bit overenthusiastic about the tū, nōs, and vōs conjugation macrons for past perfect, but future perfect lacks them, as you say